clustered/clustered-dist/src/assemble/server/conf/tc-config.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<tc-config xmlns="http://www.terracotta.org/config"
+           xmlns:ohr="http://www.terracotta.org/config/offheap-resource">
+  <!--
+    This is the default Terracotta server configuration file for the Ehcache kit.
+
+    It defines a single offheap resource of 512MB to be used for caching data.
+
+    It also defines a single server, but you can add another one to benefit from high availability.
+  -->
+
+  <plugins>
+    <config>
+      <ohr:offheap-resources>
+        <ohr:resource name="main" unit="MB">512</ohr:resource>
+        <!-- You can define extra offheap resource if you want isolated resource -->
+        <!--<ohr:resource name="other" unit="MB">128</ohr:resource>-->
```

Comment:
Ehcache has indeed client side features to handle that - removed comment and commented XML.